terraform/security_groups.tf

```diff
@@ -0,0 +1,88 @@
+
+resource "aws_instance" "db-srv" {
```

Comment:
It's OK, but if the file named "security_groups", we can logically wait db instance configuration in separate file.